src/main/java/core/basesyntax/model/BalanceOperation.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.model;
```

Comment:
```suggestion
package core.basesyntax.strategy;
```
same for all other operations